include/DataFile.h

```diff
@@ -128,6 +128,7 @@ class LMMS_EXPORT DataFile : public QDomDocument
 	void upgrade_bbTcoRename();
 	void upgrade_sampleAndHold();
 	void upgrade_midiCCIndexing();
+  void upgrade_noteTypes();
```

Comment:
Fix indentation using tabs

Reply:
My bad, I used Github editor for this fix and I think it converts tabs to spaces. I even copied the indenting from the line above to make sure but it still left spaces :sweat_smile: